packages/donor/src/screens/unsubscribe/UnsubscribedScreen.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { MainNavigationKeys } from "../../navigation/app/MainNavigationKeys";
+import Button, { ButtonVariant } from "../../components/basic/Button";
+import { useNavigate } from "react-router-dom";
+import zmDrop from "../../assets/images/zmDrop.svg";
+import styles from "./UnsubscribedScreen.module.scss";
+
+const UnsubscribedScreen = () => {
+  const navigate = useNavigate();
+
+  const profileLink = (
+    <Button
+      onClick={() => navigate(MainNavigationKeys.MyProfile)}
+      title={"הפרופיל"}
+      variant={ButtonVariant.text}
+      className={styles.ProfileLink}
```

Comment:
Added the Button but I think that a common component for link is an overkill because : 
1. Links color (or any other attribute) shouldn't change , it is a global convention
2. Noticed hoe much code I needed to add for that : `useHistory`, `Button` imports, and styling, while with `Link` we only had to add one simple line
@yaron1m WDYT ?

Reply:
I think there's no one right way.

I understand your arguments, and you are not wrong.

Both ways look OK to me, I just want us to be consistent.